var/spack/repos/builtin/packages/linux-pam/package.py

```diff
@@ -27,6 +27,11 @@ class LinuxPam(AutotoolsPackage):
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
 
+    def flag_handler(self, name, flags):
+        if name == "ldflags":
+            flags += ["-lintl"]  # Addresses https://github.com/spack/spack/issues/44637
```

Comment:
```suggestion
            flags.append("-lintl") # spack/spack#44637
```

I'd recommend simply appending the flag here instead of creating a new list to then merge it with flags.